src/java/org/apache/cassandra/db/compaction/OperationType.java

```diff
@@ -50,15 +50,35 @@
     VIEW_BUILD("View build"),
     INDEX_SUMMARY("Index summary redistribution"),
     RELOCATE("Relocate sstables to correct disk"),
-    GARBAGE_COLLECT("Remove deleted data");
+    GARBAGE_COLLECT("Remove deleted data"),
+    DOWNGRADE_SSTABLES("Downgrade sstables"),
+    RESTORE("Restore"),
+    // operations used for sstables on remote storage
+    REMOTE_RELOAD("Remote reload", true), // reload locally sstables that already exist remotely
+    REMOTE_COMPACTION("Remote compaction", true), // no longer used, kept for backward compatibility
```

Comment:
this one is not being used. does it make sense to keep it (upgrades etc)?

Reply:
Let's keep it for upgrade. In case there are some very old compactors